examples/inference/run_llama_inference.py

```diff
@@ -21,18 +20,24 @@ def run_inference(args):
     pp_size = args.pp_size
     rank = dist.get_rank()
 
-    tokenizer = LlamaTokenizer.from_pretrained(llama_model_path)
-    tokenizer.pad_token_id = tokenizer.unk_token_id
-    model = LlamaForCausalLM.from_pretrained(llama_model_path, pad_token_id=tokenizer.eos_token_id)
-    model = model.half()
+    if args.quant is None:
+        tokenizer = LlamaTokenizer.from_pretrained(llama_model_path)
+        tokenizer.pad_token_id = tokenizer.unk_token_id
+        model = LlamaForCausalLM.from_pretrained(llama_model_path, pad_token_id=tokenizer.eos_token_id)
```

Comment:
This is strange. You set `pad_token_id=unk_token_id` for tokenzier, but `pad_token_id=eos_token_id` for model

Reply:
Updated.